frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -258,6 +282,37 @@
           <template #title><div></div></template>
           <template #runs><div></div></template>
         </omegaup-arena-runs>
+        <omegaup-arena-runs-for-courses
+          v-if="useNewVerdictTable"
```

Comment:
Acá también funcionaría el `v-else`
```suggestion
          v-else
```
?